src/objectTranslations/Get_Started_Completed_Checklist__c-nl_NL.objectTranslation

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <plural>false</plural>
+        <value>Getting Started Completed Checklist</value>
```

Comment:
@dcaballeroSf  Shouldn't these be commented out vs. showing as an actual translation? How are you generating the translation files?

Reply:
I recreated them manually. I couldnt find an efficient way to generate them manually(object is not showing in Translation Workbench), so it was easier to me to use another one to adapt to my object. I commented out this line anyway

Reply:
The best way to do this, not that I'm asking you to go through the process at this point, is to use `cci task run retrieve` to pull down the translation files from the scratch org.